inc/zoo/swar/SWAR.h

```diff
@@ -138,9 +140,12 @@ template<int NBits, typename T = uint64_t> struct SWAR {
         return SWAR(m_v & mask);
     }
 
-    /// Notably, this is the SWAR lane index that contains the MSB.
+    /// The SWAR lane index that contains the MSB.  It is not the bit index of
+    /// the underlying type that contains the MSB.
+    /// IE: 8 bit wide 32 bit SWAR: 0x0040'0000 will return 5, not 43.
     constexpr int top() const noexcept { return msb_index(m_v) / NBits; }
-    /// Notably, this is the SWAR lane index that contains the lsb.
+    /// The SWAR lane index that contains the LSB.  It is not the bit index of
+    /// the underlying type that contains the LSB.
```

Comment:
1. I prefer to never copy and paste, this extends to comments.
2. Alas, the comment is misleading, since here are two possible understandings: index in bits or in elements ("lanes"), the trailing part of "of the underlying type...." is unnecessary and its presence is confusing